http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/ProxyDAO.scala

```diff
@@ -43,11 +40,10 @@ object Proxy {
   ): F[HostStatus] =
     runtimeDnsCache
       .getHostStatus(RuntimeDnsCacheKey(cloudContext, runtimeName))
-      .timeout(10 seconds)
 
   def getAppTargetHost[F[_]: Async](kubernetesDnsCache: KubernetesDnsCache[F],
                                     cloudContext: CloudContext,
                                     appName: AppName
   ): F[HostStatus] =
-    kubernetesDnsCache.getHostStatus(KubernetesDnsCacheKey(cloudContext, appName)).timeout(10 seconds)
+    kubernetesDnsCache.getHostStatus(KubernetesDnsCacheKey(cloudContext, appName))
```

Comment:
I think this is still useful guard, but maybe we can respond to the timeout a bit better in leo

Reply:
My concern is if timing this out leaves us with a permanent cache miss/aborted query loop. I'd advocate for removing the timeout and accepting that a slow query execution will at least populate the cache.

Reply:
@rtitle and @Qi77Qi , is there a way we could see how many cache misses are going to the database at once? Or would we need to add totally new metrics to determine this?